src/App.tsx

```diff
@@ -170,11 +173,9 @@ const Providers: React.FC = ({ children }) => {
                                                     <BedTokenMarketDataProvider>
                                                       <BedIndexComponentsProvider>
                                                         <IndexTokenMarketDataProvider>
-                                                          <SnapshotProposalsProvider>
```

Comment:
[question] does it make sense to delete the `<SnapshotProposalsProvider>` given we're no longer using it?

Reply:
yes, @tytarman actually deleted it here

Reply:
Sorry for not being, clear. I'm proposing deleting https://github.com/SetProtocol/index-ui/blob/cfcdcbc23966e2afa13dc0868a72eb78e7f247f8/src/contexts/SnapshotProposals/SnapshotProposalsProvider.tsx#L5-L25 and any other types, interfaces, files that are no longer used.

Reply:
🤦 of course I misunderstood that, yes I think we should. I'll make an issue for it